test/unit/test_multipart.py

```diff
@@ -95,18 +94,10 @@ def test_init_defaults(self):
         generator = ipfsApi.multipart.BodyGenerator(name)
         self.assertEqual(generator.headers['Content-Disposition'],
                          expected_disposition)
-        if six.PY2:
-            if not re.match(expected_type, generator.headers['Content-Type']):
-                self.fail('Content-Type does not match "%s", got %s'
-                          % (expected_type, generator.headers['Content-Type']))
-            if not re.match(expected_boundary_pattern, generator.boundary):
-                self.fail('Boundary does not match "%s"'
-                          % expected_boundary_pattern)
-        else:
-            self.assertRegexpMatches(generator.headers['Content-Type'],
-                                     expected_type)
-            self.assertRegexpMatches(generator.boundary,
-                                     expected_boundary_pattern)
+        self.assertRegexpMatches(generator.headers['Content-Type'],
```

Comment:
It feels so much better to lose the 2.6 special cases. These tests are all cleaner and better. It makes my soul happy.